test/hummingbot/strategy/avellaneda_market_making/test_avellaneda_market_making.py

```diff
@@ -514,90 +603,44 @@ def test_calculate_target_inventory(self):
 
         self.assertEqual(expected_quantize_order_amount, self.strategy.calculate_target_inventory())
 
-    def test_get_min_and_max_spread(self):
-        # Simulate low volatility. vol approx. 0.5%
-        self.simulate_low_volatility(self.strategy)
+    def test_liquidity_estimation(self):
 
-        # Calculating min and max spread in low volatility
-        curr_price: Decimal = self.strategy.get_price()
-        expected_min_spread: Decimal = self.min_spread * curr_price
-        expected_max_spread: Decimal = self.max_spread * curr_price * (expected_min_spread / (self.min_spread * curr_price))
+        # Simulate high liquidity
+        self.simulate_high_liquidity(self.strategy, self.low_vol)
 
-        self.assertEqual((expected_min_spread, expected_max_spread), self.strategy._get_min_and_max_spread())
+        alpha, kappa = self.strategy.trading_intensity.current_value
 
-        # Simulate high volatility. vol approx. 10%
-        self.simulate_high_volatility(self.strategy)
-
-        # Initialize strategy with high vol_to_spread_multiplier config
-        self.strategy.vol_to_spread_multiplier = Decimal("10")
-
-        curr_price: Decimal = self.strategy.get_price()
-        curr_vol: Decimal = self.strategy.get_volatility()
-        expected_min_spread: Decimal = self.strategy.vol_to_spread_multiplier * curr_vol
-        expected_max_spread: Decimal = self.max_spread * curr_price * (expected_min_spread / (self.min_spread * curr_price))
+        self.assertAlmostEqual(100.58688318883304, alpha, 5)
+        self.assertAlmostEqual(0.03153874736059721, kappa, 5)
```

Comment:
It would be useful for developers in general to have a calculation here indicating how we get the expected alpha and kappa, instead of just hardcoding the expected numbers in the test.